sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -14,9 +14,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>clearFault</EFDB_Name>
-            <Description></Description>
+            <Description>Needed by SAL</Description>
```

Comment:
Standard is:

*Ignored Attributes in Topics*

Many generic commands have an ignored attribute. This is due to a requirement from the API to not have empty topics. If you are adding a command to your CSC that does not require an attribute, it still must contain a "dummy" one. In order to maintain consistency across this use case, the attribute must be called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused." and have the following units: n/a (pending verification).

https://confluence.lsstcorp.org/display/SYSENG/SAL+constraints+and+recommendations

Reply:
https://ts-xml.lsst.io/sal_constraints_and_recommendations.html#ignored-attributes-in-topics